plugin/coin_policy_manager.go

```diff
@@ -41,7 +38,7 @@ func NewCoinPolicyManager(pvm *PluginVM, state loomchain.State) (*CoinPolicyMana
 	}, nil
 }
 
-//MintCoins method of coin_deflation_Manager will be called from Block
-func (c *CoinPolicyManager) MintCoins() error {
-	  return coin.Mint(c.ctx)
+//ApplyPolicy method of coin_deflation_Manager will be called from Block
```

Comment:
-> `// ApplyPolicy will apply the current economic policy within the coin contract`

Reply:
Done.